action.yml

```diff
@@ -14,6 +14,14 @@ inputs:
   anaconda_nightly_upload_token:
     description: 'Token to upload to scientific python org'
     required: true
+  anaconda_nightly_upload_url:
+    description: 'URL to upload to scientific python org'
+    required: false
+    default: scientific-python-nightly-wheels
+  anaconda_nightly_upload_labels:
+    description: 'Labels assigned to the uploaded artifacts'
+    required: false
+    default: main
```

Comment:
Why is the label needed? This is a change in current behavior.

Reply:
By default, the label is main when you upload an artifact in anaconda

![image](https://github.com/scientific-python/upload-nightly-action/assets/23106443/30a93689-5fe2-4174-9eda-231d636551e3)

in our case (https://anaconda.org/dipy/dipy), the organization name is not explicit. I would like to avoid random people downloading nightly wheels and believing that is it a release. So, we tag them as dev instead of main (see the difference between the sreenshot above and below

![image](https://github.com/scientific-python/upload-nightly-action/assets/23106443/bef99917-8db9-48b1-8437-fe1340378de1)

one artifact can have one or multiple labels. I am fine for alternative solution to avoid confusion with users (since I can relabel all the old artifacts)

Documentation on anaconda label: https://docs.anaconda.com/free/anacondaorg/user-guide/work-with-labels/